### ISSUE TITLE
Add a push button to VOR1 to toggle CDI between GPS and NAV1

### ISSUE THREAD

**Issue by emxsys** [scott-vincent/instrument-panel#21]
The new VOR1 CDI can be toggled between the GPS cross track (default) or the NAV1 radial error.  Adding a push button to the VOR1 to perform the toggle seems like a natural extension of the rotary knob used for the OBS.  Currently, the toggle must be performed in the FS2020 UI.

**Comment by scott-vincent** [comment]
Hi Bruce,

I'm not sure VOR1 is working correctly for an ILS approach anymore. The
needle used to show me the offset to the ILS even before I had pressed LOC
to capture it but now the instrument shows GPS and the needle doesn't move
(until I press LOC). I must admit, I'm not sure how to use the GPS mode, I
only ever do ILS approaches.

Thanks,
Scott

**Comment by emxsys** [comment]
Hi Scott, 
What plane and approach are you flying? I'd like to see if I can replicate. 

Also, can you verify whether the sim's VOR behavior is in or out of sync with the panel's?

Lastly, there are a number of GPS-based flags that I haven't investigated yet that may be applicable for a GPS approach. I'll look into that.

**Comment by scott-vincent** [comment]
I was flying the A320neo into Gatwick ILS 23R with Nav1 frequency 109.50
MHz. Good point, I didn't check to see what the actual plane was showing,
silly me. I will do that now.

Note that I'm using the A32nx FlyByWire mod for the A320 as the normal A320
was broken by Asobo and hasn't been fixed yet.

Thanks,
Scott

**Comment by scott-vincent** [comment]
Did I say Gatwick, I meant Manchester. It was from Gatwick to Manchester.

**Comment by emxsys** [comment]
Ok, I will investigate.  I was testing with the C172 (with the Garmin 430) and I was flying in and out KOXR (Oxnard, CA, USA, where I used to work) with an ILS approach of 108.7. I guess it's time to learn to fly an A320.

BTW: I learned to fly out of KSZP (Santa Paula, CA, USA) and all the neighboring airports.  What's your "home" airport?

**Comment by scott-vincent** [comment]
Maybe a simple fix is to make navHasLocalizer take precedence over
gpsDrivesNav1. Wouldn't it behave like it did before then? I'm used to it
showing the ILS needle once I tune Nav1 to the ILS frequency.

**Comment by scott-vincent** [comment]
Yep, seems to work. If I tune NAV1 to the ILS frequency and the ILS is in
range then the variable simVars->navHasLoc goes from 0 to 1.

**Comment by emxsys** [comment]
Great discovery! And interesting nonetheless. I'll start investigation on the GPS approach types to ensure we can have a comprehensive solution.
